src/widget/city_without_overlay.c

```diff
@@ -471,6 +471,77 @@ static void draw_plague(building *b, int x, int y, color_t color_mask)
     }
 }
 
+static void draw_depot_ressource(building *b, int x, int y, color_t color_mask)
```

Comment:
"resource" only has one S :stuck_out_tongue:

Reply:
Sorry there are 2 s in french...